### ISSUE TITLE
Cannot get required symbol TLSv1_1_method from libssl

### ISSUE THREAD

**Issue by livarcocc** [dotnet/core-setup#3333]
_From @jafg1222 on October 22, 2017 0:13_

## Steps to reproduce
`dotnet new console -o hwapp`  
## Expected  behavior
Create a new application with dotnet cli
## Actual behavior
`Cannot get required symbol TLSv1_1_method from libssl
Aborted (`core' generado)`

## Environment data
`dotnet --info` output:
Herramientas de la línea de comandos .NET (2.0.0)

Product Information:
 Version:            2.0.0
 Commit SHA-1 hash:  cdcd1928c9

Runtime Environment:
 OS Name:     fedora
 OS Version:  26
 OS Platform: Linux
 RID:         fedora.26-x64
 Base Path:   /usr/share/dotnet/sdk/2.0.0/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d

_Copied from original issue: dotnet/cli#7882_

**Comment by livarcocc** [comment]
Seems like a native requirement issue. Moving to core-setup.

**Comment by rakeshsinghranchi** [comment]
System dependencies are missing . Please follow below steps : https://www.microsoft.com/net/core#linuxfedora 
 sudo dnf update
 sudo dnf install libunwind libicu compat-openssl10

**Comment by rakeshsinghranchi** [comment]
Closing the issue as this appears to be missing system dependencies.

**Comment by landerop** [comment]
I'm running a Fedora 24, x64. I've tried to run this command today and I'm still getting the error. I have all dependencies installed and I've followed steps you put. 
Any idea?

![error](https://user-images.githubusercontent.com/8417930/45375440-31da5b00-b5b2-11e8-9c05-9b2372be1f2e.png)
![dependencies](https://user-images.githubusercontent.com/8417930/45375442-31da5b00-b5b2-11e8-8605-774a5d0a1843.png)
